packages/injected/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@web3-onboard/injected-wallets",
-  "version": "2.3.0",
+  "version": "2.3.1",
```

```suggestion
  "version": "2.3.1-alpha.1",
```
This will allow for alpha version publish for testing prior to production release